packages/hash/frontend/src/components/hooks/blockProtocolFunctions/ontology/useBlockProtocolGetDataType.ts

```diff
@@ -32,8 +32,11 @@ export const useBlockProtocolGetDataType = (): {
         };
       }
 
+      const { dataTypeId } = data;
+
       const response = await getFn({
         query: getDataTypeQuery,
+        variables: { dataTypeId },
       });
```

Comment:
These are drive-by fixes for the existing `useBlockProtocolGetDataType`/`EntityType`/`PropertyType`/`LinkType` hooks, which to my understanding don't work in the current implementation because they aren't passing required variables to their respective queries